docs/build_docs.sh

```diff
@@ -24,5 +24,6 @@ then
 	exit 1
 fi
 git submodule update --init --recursive
-
+source ./setup_docs.sh
```

Comment:
is there a particular reason why this one uses `source`?

Reply:
No, I just searched if `source` was used elsewhere in the Flink codebase and saw it was, so used that. Any preference from your end?

Reply:
🤷 I'd say just running the script would be cleaner; in my mind source is a sort of import statement.